build/makelib/golang.mk

```diff
@@ -152,7 +152,8 @@ go.lint: $(GOLINT)
 .PHONY: go.vet
 go.vet:
 	@echo === go vet
-	@CGO_ENABLED=0 $(GOHOST) vet $(GO_COMMON_FLAGS) $(GO_PACKAGES) $(GO_INTEGRATION_TEST_PACKAGES)
+	@echo CGO_ENABLED=$(CGO_ENABLED_VALUE) $(GOHOST) vet $(GO_COMMON_FLAGS) $(GO_PACKAGES) $(GO_INTEGRATION_TEST_PACKAGES)
```

Comment:
Should this echo be removed, or useful to keep?

Reply:
I've it to be useful to see the args we use when running.

Reply:
Alternately, we can just remove the `@` before the line below, and then we don't need this echo. ;)

Reply:
Rah I knew we had discussed that earlier! Thanks for the reminder.